src/backend/api/Fusion.Resources.Api/Controllers/Requests/ApiModels/ApiProposedPerson.cs

```diff
@@ -18,13 +20,17 @@ public ApiProposedPerson(QueryResourceAllocationRequest.QueryProposedPerson prop
             {
                 ResourceOwner = new ApiPerson(proposedPerson.ResourceOwner);
             }
-        }
 
+            if (proposedPerson.DelegatedResourceOwners is not null)
+            {
+                DelegatedResourceOwners = new List<FusionPersonProfile?>(proposedPerson.DelegatedResourceOwners);
```

Comment:
given update of property, this must be changed to something like:
DelegatedResourceOwners = proposedPerson.DelegatedResourceOwners.Select(x => new ApiPerson(x));